testing/describes.js

```diff
@@ -648,10 +648,7 @@ class RealWinFixture {
         } else {
           // The anonymous class parameter allows us to detect native classes
           // vs transpiled classes.
-          installCustomElements(
-            win,
-            NATIVE_CUSTOM_ELEMENTS_V1 ? class {} : undefined
-          );
+          installCustomElements(win, class {});
```

Comment:
We always transpile our testing code right? Did we try running our test suite against the native CE code path?

Reply:
I'm not sure if the ESM builds ran with this RTV experiment enabled. @erwinmombay?